SilKit/cmake/qnx-cross-base-toolchain.cmake

```
@@ -73,7 +73,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Y_cxx -stdlib=libc++")
 set(CMAKE_EXE_LINKER_FLAGS "-Wl,-z,origin")
 
 if($ENV{QNX_HOST} MATCHES ".*qnx710.*")
-    message(STATUS "QNX700 detected: Linking against libc++fs")
+    message(STATUS "QNX710 detected: Linking against libc++fs")
     link_libraries(-lc++fs)
```

You could try it with `set` and the `CMAKE_CXX_STANDARD_LIBRARIES` variable? This is likely cleaner and 'safer' than using the directory-scoped `link_libraries(...)` function.

```suggestion
    list(APPEND CMAKE_CXX_STANDARD_LIBRARIES "c++fs")
```